Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -98,6 +98,7 @@
     <Compile Include="**\*.ios.*.cs" />
     <Reference Include="System.Numerics" />
     <Reference Include="System.Numerics.Vectors" />
+    <Reference Include="OpenTK-1.0" />
```

Comment:
How does this affect the generated .nuspec? Is it added as a correct reference?

Reply:
It will automatically figure it out :)